src/main/java/mate/academy/rickandmorty/repository/CharacterRepository.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.rickandmorty.repository;
+
+import java.util.List;
+import mate.academy.rickandmorty.model.Character;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface CharacterRepository extends JpaRepository<Character, Long> {
+    @Query("SELECT c FROM Character c ORDER BY RAND() LIMIT 1")
```

Comment:
```suggestion
    @Query("SELECT c FROM Character c ORDER BY RAND() LIMIT 1")
```
It is not really getting the random character...